### ISSUE TITLE
Pa11y won't exit with error when it fails to launch the browser

### ISSUE THREAD

**Issue by bfabio** [italia/developers.italia.it#891]
Looks like pa11y doesn't exit with error if there are problems when launching the browser for the accessibility tests, resulting in a successful pipeline when it should be a failed one.
